virtwho/ssh.py

```diff
@@ -1,3 +1,152 @@
-"""
-Define the paramiko ssh/sftp
-"""
+import os
+import paramiko
+from six import BytesIO
```

Comment:
It's my fault, there is no BytesIO required for this file.

Reply:
Removed it.